templates/index.html

```
@@ -1,145 +1,198 @@
-<!-- # Copyright (c) Liam Suorsa and Mika Suorsa -->
-{% extends 'base.html' %}
+<!-- # Copyright (c) Liam Suorsa and Mika Suorsa -->
+{% extends 'base.html' %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)